sulong/projects/com.oracle.truffle.llvm.nodes/src/com/oracle/truffle/llvm/nodes/intrinsics/rust/LLVMStart.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
I just noticed the file mode change. Is that intentional? I don't think source files should ever be executable...

Reply:
Sorry, the file mod change was completely unintentional. I fixed that.